cockatrice/src/tab_game.cpp

```diff
@@ -837,6 +837,8 @@ void TabGame::sendGameCommand(PendingCommand *pend, int playerId)
     AbstractClient *client = getClientForPlayer(playerId);
     if (!client)
         return;
+
+    connect(pend, SIGNAL(finished(Response, CommandContainer, QVariant)), this, SLOT(commandFinished(const Response &)));
```

Comment:
in TabGame there are two different sendGameCommand implementations, the first taking a PendingCommand* parameter and the second taking a google::protobuf::Message& parameter. They are called from different places, but they never call one eachother.